src/rule-no-future-added-value.js

```diff
@@ -31,7 +21,10 @@ module.exports = context => {
     }
   }
   return {
-    Literal: validateValue,
-    StringLiteralTypeAnnotation: validateValue
+    "Literal[value='%future added value']": validateValue,
+
+    // StringLiteralTypeAnnotations that are not children of a default case
+    [':not(SwitchCase[test=null] StringLiteralTypeAnnotation)' +
```

Comment:
Woah, what is this sorcery!

I suppose we could even delete the condition on line 14?

Reply:
Ya it could be removed as well I think!

Only thing is I'm not sure about the performance of the second selector (in a huge repo), but the only other way to check for the parent being a default case is `context.getAncestors()`